pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -31,11 +33,20 @@ public class ElasticSearchParticipantDto {
     private List<Map<String, Object>> workflows;
     private String status;
     private ESDsm dsm;
+    @Getter
     private String ddp;
 
+    @Getter
+    private Map<String, Object> dataAsMap;
```

Comment:
why it became necessary to add this property? I believe `ElasticSearchParticipantDto` already holds extracted data from source map.

Reply:
correct, it holds the *extracted* data.  But I need a place to store the raw data, so that I can skip the extraction step.

Reply:
could you please provide more details why it became necessary to keep raw data separately?